src/sentry/stacktraces/processing.py

```diff
@@ -255,8 +257,26 @@ def normalize_stacktraces_for_grouping(data, grouping_config=None):
     if not stacktraces:
         return
 
-    # If a grouping config is available, run grouping enhancers
     platform = data.get('platform')
+
+    # Put the trimmed function names into the frames.  We only do this if
+    # the trimming produces a different function than the function we have
+    # otherwise stored in `function` to not make the payload larger
+    # unnecessarily.
+    for frames in stacktraces:
+        for frame in frames:
+            if frame.get('raw_function') is not None:
+                continue
```

Comment:
This means that SDKs cannot ingest just a `raw_function`. In processing, the raw function name is only trimmed if there is a symbolication match. There's also no normalization that moves ingested function names to `raw_function` for this case.

Reply:
Per policy you cannot send *just* `raw_function`. That is only permitted if you also send `function`.